content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -7,7 +7,7 @@ tocDepth: 2
 
 <TopBlock>
 
-The data model definition part of the [Prisma schema](../) defines your application models (also called **Prisma models**). Models:
+The data model definition part of the [Prisma schema](./) defines your application models (also called **Prisma models**). Models:
```

Comment:
Interesting, that is indeed wrong right now on https://www.prisma.io/docs/concepts/components/prisma-schema/data-model 👍